src/utils/file-upload-utils.js

```diff
@@ -44,11 +44,16 @@ const scanFileForVirus = (fileBuffer, timeout) => {
     defaultCloudmersiveClient.timeout = timeout
   }
   return new Promise((success, failure) => {
-    apiInstance.scanFile(fileBuffer, (error, data) => {
+    apiInstance.scanFile(fileBuffer, (error, data, response) => {
       if (error) {
-        logger.error(
-          `Error when calling Cloudmersive Virus Scan API: ${error.message}`
-        )
+        logger.error({
+          message: "Error when calling Cloudmersive Virus Scan API",
+          error,
+          meta: {
+            data,
```

Comment:
Just to note: I'm not sure if `data` will be populated on error (as opposed to being `null`/`undefined`, and we'd have to retrieve the response body from the response object ourselves) 🤔 

But I reckon it's not very important just yet, this PR already logs a lot more info that we had before, and we can iterate to get the body later.